addons/account/models/account_journal_dashboard.py

```diff
@@ -508,7 +511,23 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
             )
         }
 
-        sale_purchase_journals._fill_dashboard_data_count(dashboard_data, 'account.move', 'entries_count', [])
+        self.env.cr.execute(SQL("""
+            SELECT id, moves_exists
+            FROM account_journal journal
+            LEFT JOIN LATERAL (
+                SELECT EXISTS(SELECT 1
+                              FROM account_move move
+                              WHERE move.journal_id = journal.id
+                              AND (move.company_id IS NULL OR (move.company_id = ANY (%(companies_ids)s)) AND
```

Comment:
`company_id` is not nullable

Reply:
Oups, my bad, I didn't check if it was required. Forgot the ORM generates the useless null check for required fields.
Pushed the correction.